### PR TITLE
Updated design based on Pycon IL

### PR DESCRIPTION
**Pull request by lilachn**
Requires review before merge!

Includes English/Hebrew support, as well as mobile support.

### CODE REVIEW

**Comment by lilachn** [comment]
@moshe742 this PR includes your changes as well as mine. Your feedback is welcome.

File: en/index.html [hunk 1]
```diff
@@ -107,12 +133,33 @@ <h1 class="noMobile">
        and comfortable environment for everybody.
       </div>
 
+      <a id="sponsors" name="sponsors"></a>
+      <h2>Sponsored by:</h2>
+      <div class="sponsors">
+        Want to sponsor? Please contact <a href="mailto:board@hamakor.org.il">board@hamakor.org.il</a>
+        <div class="diamond-rtl noMobile">
+          <h3>Diamond Sponsers</h3>
```
Comment (tbrisker) [inline en/index.html:141]: Gold.

File: en/index.html [hunk 2]
```diff
@@ -107,12 +133,33 @@ <h1 class="noMobile">
...
+          <div class="sponsor">
+            <a href="https://www.redhat.com/en/global/israel ">
+              <img src="../sponsor_logos/Redhat.png">
+            </a>
+          </div>
```
Comment (tbrisker) [inline en/index.html:146]: please add the other commit levels (silver, bronze)

File: en/index.html [hunk 3]
```diff
@@ -107,12 +133,33 @@ <h1 class="noMobile">
...
+        </div>
+      </div>
+
       <div id="greetings">Looking forward to see you!</div>
 
     </div>
 
     <div id="footer">
-      cc-by-sa <a href="http://hamakor.org.il">Hamakor</a>. Design by <a href="http://tomercohen.com">Tomer Cohen</a>, graphics by <a href="http://sijp.co.il">Shlomi Israely</a>, font by <a href="http://culmus.sf.net">Culmus project</a>.
+      cc-by-sa <a href="http://hamakor.org.il">Hamakor</a>. Created by <a href="http://tomercohen.com">Tomer Cohen</a>, graphics by <a href="http://sijp.co.il">Shlomi Israely</a>, font by <a href="http://culmus.sf.net">Culmus project</a>.
```
Comment (tbrisker) [inline en/index.html:155]: is this still relevant? other then the licence part?

File: index.html [hunk 4]
```diff
@@ -24,116 +24,152 @@
...
-        <h1>August Penguin 2016</h1>
+    <div class="container">
+        <header>
+            <nav>
+                <ul>
+                    <li>
+                        <a href="index.html">אודות</a>
+                    </li>
+                    <li>
+                        <a href="#schedule">הרצאות/מרצים</a>
```
Comment (tbrisker) [inline index.html:35]: לו"ז הכנס

File: index.html [hunk 5]
```diff
@@ -24,116 +24,152 @@
...
+                    </li>
+                    <li>
+                        <a href="code_of_conduct.html">כללי התנהגות</a>
+                    </li>
+                    <li>
+                        <a href="en/">English</a>
+                    </li>
```
Comment (tbrisker) [inline index.html:42]: כדאי להוסיף לינק להרשמה גם

File: en/index.html [hunk 6]
```diff
@@ -25,26 +25,54 @@
...
+        <nav>
+          <ul>
+            <li>
+              <a href="index.html">About</a>
+            </li>
+            <li>
+              <a href="#schedule">Agenda</a>
+            </li>
+            <li>
+              <a href="code_of_conduct.html">Code of Conduct</a>
```
Comment (tbrisker) [inline en/index.html:39]: we should also add the header in the code of conduct or open the code of conduct in a separate tab

**Comment by tbrisker** [comment]
closing in favor https://github.com/Hamakor/AugustPenguin/pull/80

**Pull request closed without merging**